vms/platformvm/state/staker_test.go

```diff
@@ -144,11 +144,11 @@ func TestNewCurrentStaker(t *testing.T) {
 	subnetID := ids.GenerateTestID()
 	weight := uint64(12345)
 	startTime := time.Now()
-	endTime := time.Now()
+	endTime := startTime.Add(time.Hour)
```

Comment:
just a nit, not strictly relevant to this PR, but I'd rather take it out of the way